WalletWasabi.Gui/Global.cs

```diff
@@ -361,6 +367,17 @@ public async Task InitializeNoWalletAsync()
 
 				WalletManager.RegisterServices(BitcoinStore, Synchronizer, Nodes, Config.ServiceConfiguration, FeeProviders, blockProvider);
 			}
+			catch (Exception ex)
+			{
+				if (!cancel.IsCancellationRequested)
+				{
+					Logger.LogCritical(ex);
+				}
+
+				InitializationCompleted = true;
+				await DisposeAsync().ConfigureAwait(false);
+				Environment.Exit(1);
+			}
```

Comment:
https://github.com/zkSNACKs/WalletWasabi/pull/3702/commits/dfd6335082a1b080035df6ee7a1fc0b2c0a4fd19 added this back.
Wasn't this part removed in #3707?

Reply:
Good catch, I ll fix.